tests/js/spec/utils/withUserTeams.spec.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import withUsersTeams from 'app/utils/withUsersTeams';
+
+describe('withUserTeams HoC', function() {
+  const api = new MockApiClient();
+  beforeEach(function() {
+    MockApiClient.clearMockResponses();
+  });
+
+  it('forwards errors', async function() {
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/user-teams/',
+      statusCode: 400,
+    });
+    const MyComponent = () => null;
+    const Container = withUsersTeams(MyComponent);
+    const wrapper = mount(<Container organization={{slug: 'org-slug'}} api={api} />);
+    await wrapper.instance().fetchTeams();
```

Comment:
Is this necessary since this gets called on `didMount`? If it's an async issue, you can `await tick()`

Reply:
gotcha, yea I was having trouble with async but await tick() works out well